.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is.
+
+**How To Reproduce**
+
+Fikes to reproduce the behavior:
```

Comment:
I'm not sure what "fikes" mean in this context... maybe it's a typo and should be "Files" instead?

I think this should be `Relevant information and input files to reproduce the behavior`, which I think it better matches what we are requesting here.

On top of that change, a nit: I'd **also change the list into a non-numbered bullet item list** (instead of ordered), since there's no need to provide that information in any particular order.